lib/commonAPI/coreapi/ext/platform/android/jni/SystemImpl.cpp

```diff
@@ -66,7 +66,7 @@ class CSystemImpl : public CSystemImplBase
     virtual void getDeviceOwnerName(rho::apiGenerator::CMethodResult& result);
     virtual void getDeviceOwnerEmail(rho::apiGenerator::CMethodResult& result);
     virtual void getOsVersion(rho::apiGenerator::CMethodResult& result);
-    virtual void getIsMotorolaDevice(rho::apiGenerator::CMethodResult& result);
```

Comment:
Hi, fyi we still need to expose isMotorolaDevice for backwards compatibility but we don't want to document it.